app/components/candidate-preamble.jsx

```diff
@@ -77,20 +78,39 @@ class CandidateJoin extends Join {
 }
 
 class CandidatePreamble extends React.Component {
+  makeQuestions = questions => {
```

Comment:
This method is hwaatt 🔥🔥🔥
Cool way you handle nested Arrays in this case

Reply:
thanks!